package-lock.json

```diff
@@ -1269,6 +1268,12 @@
         "http-errors": "~1.7.2"
       },
       "dependencies": {
+        "deep-equal": {
```

Comment:
Not sure why this is appearing here as I removed it from package.json and did a clean install. It shows as a transitive dependency as well...